pisa/analysis/README.md

```diff
@@ -2,4 +2,5 @@
 
 Primary tools for performing analyses in PISA.
 
-recent changes in `analysis.py` see https://nbviewer.jupyter.org/urls/seafile.rlp.net/f/c156d1fc084f4bd8af63//%3Fdl%3D1
+Documentation page for the PISA analysis (analysis.py),
+https://user-web.icecube.wisc.edu/~peller/pisa_docs/pisa.analysis.html?highlight=analysis%20analysis#module-pisa.analysis.analysis
```

Comment:
Our documentation is now officially located at https://icecube.github.io/pisa/, so please use that url instead.